amazon-kinesis-client/src/main/java/software/amazon/kinesis/leases/dynamodb/DynamoDBLeaseRefresher.java

```diff
@@ -162,16 +162,7 @@ public DynamoDBLeaseRefresher(final String table, final DynamoDbAsyncClient dyna
     @Override
     public boolean createLeaseTableIfNotExists(@NonNull final Long readCapacity, @NonNull final Long writeCapacity)
             throws ProvisionedThroughputException, DependencyException {
-        try {
```

Comment:
Why are we deleting this block of code? This also changes customer-facing behavior - let's remove this from the diff.